src/electron/platform/android/setup/android-port-cleaner.ts

```diff
@@ -37,7 +37,6 @@ export class AndroidPortCleaner {
                 if (p) {
                     try {
                         await this.serviceConfig.removeTcpForwarding(p);
-                        await this.serviceConfig.listForwardedPorts(); // Allows ADB to complete
```

Comment:
Why did you remove the call to list the forwarded ports here?

Reply:
good question - the comment & prior PR show it was intended to try helping ADB commands fully complete before the app closed, but it turns out this behavior was due to our missing async listeners & not because of appium itself; we don't need the list for the production flow